Workbooks/Azure Active Directory/SignInsUsingLegacyAuth/SignIns.workbook

```diff
@@ -206,7 +207,7 @@
     errorCode == 50140, \"Interrupt\",                       \r\n                            errorCode == 51006, \"Interrupt\",                       \r\n                            errorCode == 50059, \"Interrupt\",                       \r\n                            errorCode == 65001, \"Interrupt\",                       \r\n                            errorCode == 52004, \"Interrupt\",                       \r\n                            errorCode == 50055, \"Interrupt\",                       \r\n                            errorCode == 50144, \"Interrupt\",                       \r\n                            errorCode == 50072, \"Interrupt\",                       \r\n                            errorCode == 50074, \"Interrupt\",                       \r\n                            errorCode == 16000, \"Interrupt\",                       \r\n                            errorCode == 16001, \"Interrupt\",                       \r\n                            errorCode == 16003, \"Interrupt\",                       \r\n                            errorCode == 50127, \"Interrupt\",                       \r\n                            errorCode == 50125, \"Interrupt\",                       \r\n                            errorCode == 50129, \"Interrupt\",                       \r\n                            errorCode == 50143, \"Interrupt\",                       \r\n                            errorCode == 81010, \"Interrupt\",                       \r\n                            errorCode == 81014, \"Interrupt\",                       \r\n                            errorCode == 81012 ,\"Interrupt\",                       \r\n                            \"Failure\") \r\n|extend Reason = tostring(Status.failureReason)\r\n|extend ClientAppUsed = iff(isempty(ClientAppUsed)==true,\"Unknown\" ,ClientAppUsed)  \r\n| where ClientAppUsed in ({Protocols}) or '*' in ({Protocols}) or  ('Unknown' in ({Protocols}) and isempty(ClientAppUsed))\r\n|extend isLegacyAuth = case(ClientAppUsed contains \"Browser\", \"No\", ClientAppUsed contains \"Mobile Apps and Desktop clients\", \"No\", ClientAppUsed contains \"Exchange ActiveSync\", \"Yes\", ClientAppUsed contains \"Unknown\", \"Unknown\", \"Yes\") \r\n|where isLegacyAuth==\"Yes\";\r\ndata\r\n| summarize Count = count() by SigninStatus\r\n| join kind = inner (data\r\n        | make-series Trend = count() default = 0 on TimeGenerated in range({TimeRange:start}, {TimeRange:end}, {TimeRange:grain}) by SigninStatus) on SigninStatus\r\n| project-away SigninStatus1, TimeGenerated\r\n| union (data\r\n        | summarize Count = count()\r\n        | extend SigninStatus = 'All Sign-ins', jkey = 1\r\n        | join kind = inner (data\r\n                            | make-series Trend = count() default = 0 on TimeGenerated in range({TimeRange:start}, {TimeRange:end}, {TimeRange:grain}) \r\n                            | extend jkey = 1) on jkey\r\n        | project-away jkey, jkey1, TimeGenerated\r\n\r\n)\r\n",
```

Comment:
It seems like you replaced Other clients with Unknown at "ClientAppUsed contains \"Unknown\", \"Unknown\""
I would advise keep both Other Clients and Unknown to make it backward compatible for the data that might have OtherClients as a value in it.

Reply:
That's not quite what's happening in that line. Here's the logic: 

isLegacyAuth:
- if client is Browser -> no
- if client is Mobile Apps and Desktop clients -> no
- if client is Exchange ActiveSync -> yes
- if client is Unknown -> unknown
- ELSE (including if client is Other clients) -> yes

Therefore, you can see that client apps that are unknown are still appearing as unknown. Everything else is considered legacy auth, including Other clients. Would you like me to add a separate case to explicitly map Other clients to return yes?